File: ejercicios/python/ejercicio085.py
__size = 5

def llenado_usuario(m):
    for i in range(__size):
        n=[]
        for j in range(__size):
            num = int(input("ingrese el valor para la posicion (" + str(i) + "," + str(j) + ") : "))
            n.append(num)
        m.append(n)

def mostrar_matriz(m):
    for i in m:
        for j in i:
            print(j, end='\t')
        print("")

if __name__ == "__main__":
    matriz=[]
    positivos = 0
    negativos = 0
    llenado_usuario(matriz)
    mostrar_matriz(matriz)
    for i in matriz:
        for j in i:
            if j > 0:
                positivos += j
            else:
                negativos += j
    
    print("positivos = " + str(positivos))
    print("negativos = " + str(negativos))

File: ejercicios/c++/ejercicio085.cpp
#include <iostream>
using namespace std;
//procedimiento que llena la matriz
void llenado(int[5][5]);

//procedimiento que muestra en pantalla la matriz
void ver_matriz(int[5][5]);

int main(int argc, char *argv[]) {
	int matriz[5][5];
	int positivos=0;
	int negativos=0;
	
	llenado(matriz);
	
	for(int f=0; f<5; f++){
		for(int c=0; c<5; c++){
			if(matriz[f][c]>0){
				positivos+=matriz[f][c];
			}else{
				negativos+=matriz[f][c];
			}
		}
	}
	
	ver_matriz(matriz);
	
	cout << "positivos = " << positivos << endl;
	cout << "negativos = " << negativos << endl;
	
	return 0;
}

void llenado(int m[5][5]){
	for(int f=0; f<5; f++){
		for(int c=0; c<5; c++){
			cout<<"ingrese el valor para la fila "<<f<<" y la columna "<<c<<" : ";
			cin>>m[f][c];
		}
	}
}

void ver_matriz(int m[5][5]){
	for(int f=0; f<5; f++){
		for(int c=0; c<5; c++){
			cout << m[f][c] << "\t";
		}
		cout << endl;
	}
}

